CraftTweaker2-API/src/main/java/crafttweaker/api/util/Math.java

```diff
@@ -0,0 +1,169 @@
+package crafttweaker.api.util;
+
+import crafttweaker.annotations.ZenRegister;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+@ZenClass("crafttweaker.util.Math")
+@ZenRegister
+public class Math {
```

Comment:
I'm fine with using `Math` as a name exposed to scripts, but I would prefer if this was named something like `MathUtil` in the java code.